apps/typegpu-docs/src/content/examples/simulation/stable-fluid/index.ts

```diff
@@ -147,7 +147,7 @@ function createRenderPipeline(
   fragmentFn: TgpuFragmentFn<{ uv: d.Vec2f }, d.Vec4f>,
 ) {
   return root['~unstable']
-    .withVertex(renderFn, renderFn.shell.attributes)
+    .withVertex(renderFn, [renderFn.shell.in])
```

Comment:
Vertex attributes from a vertex layout should be passed in here, which in this case is just `{}` (no attributes). The original code was also erroneous.
```suggestion
    .withVertex(renderFn, {})
```